internal/handlers/pg/pgdb/transactions.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 // maxRetries is the maximum number of times to retry a transaction.
-const maxRetries = 3
+const maxRetries = 20
```

Comment:
The tricky part: this number of retries is enough for our tests (e.g. on my local machine there are ~66 parallel collections created), but if there are 100 or 1000 collections that are created in parallel, this number won't be enough.

Reply:
probably we will need a bit smarter retry strategy like exponential back off if we find 20 is too small.